corehq/apps/formplayer_api/smsforms/api.py

```diff
@@ -373,11 +360,10 @@ def get_raw_instance(session_id, domain=None, auth=None):
         }
 
     response = post_data(json.dumps(data), auth)
-    response = json.loads(response)
     if "error" in response:
         error = response["error"]
-        if error == "invalid session id":
```

Comment:
Based on where this was being returned from formplayer I don't think it was correct in the first place.